ear-production-suite-plugins/plugins/direct_speakers/src/direct_speakers_plugin_editor.cpp

```diff
@@ -41,8 +41,8 @@ DirectSpeakersAudioProcessorEditor::DirectSpeakersAudioProcessorEditor(
   addAndMakeVisible(viewport_.get());
 
   setResizable(true, false);
-  setResizeLimits(0, 0, 726, 930);
-  setSize(726, 930);
+  setResizeLimits(0, 0, 750, 930);
+  setSize(750, 930);
```

Comment:
Much less bothered about magic numbers here from a readability viewpoint as it feels a lot more obvious from the context what these are for and anyone searching for 'size' is going to find them, but given they're used in two places I'd probably do it anyway.

Reply:
In this case, should I update the other plugins too?

Reply:
I leave the Scene as it is since the size and resize limits are different. Also the monitoring plugins as they're not resizable.